src/main/plugin/iso19139.ca.HNAP/convert/to19139.xsl

```diff
@@ -26,13 +26,16 @@
                 xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
                 exclude-result-prefixes="#all">
 
+  <xsl:import href="../update-fixed-info.xsl"/>
```

Comment:
I believe that by by importing the update-fixed-info.xsl you will be applying all the templates within.  
I'm not sure if we want to be applying all those fixes during the export.

i.e. the update-fixed-info.xsl will be applying a gmd:locale for both the main language and the alternate language.  This is actually something that we don't want in the export as older geonetwork (i.e. FGP) seem to have issues with them. Does not seem to be an issue with the newer GN.

Reply:
Maybe you can move "add-namespaces" template to src/main/plugin/iso19139.ca.HNAP/convert/functions.xsl
And then you can include src/main/plugin/iso19139.ca.HNAP/convert/functions.xsl from both update-fixed-info.xsl and to19139.xsl.
The "add-namespaces" is also used in iso19139.ca.HNAP.FGP-to-iso19139.ca.HNAP.xsl so it can probably be updated as well.

Reply:
I have moved the add-namespace and its related variables to function.xsl as common template. Then include it to fromISO19139, to19139 and update-fixed-info.xsl. It's still working as import and export and code looks much cleaner.